02-Reproject-Raster-In-R.Rmd

```diff
@@ -83,18 +91,23 @@ This lesson is a part of a series of raster data in R lessons:
 * [Lesson 06 - Plot Raster Time Series Data in R Using RasterVis and LevelPlot]({{ site.baseurl}}/R/Plot-Raster-Times-Series-Data-In-R/)
```

Comment:
I can't decide if in the list they should be removed or not.  I did in lesson 00 but then decided maybe it is nice...    In the tabular time series, we now have the 07: Convert to Julian Day lesson.  It is really an additional information lesson, not the 7th in a series.  This is one of my issues with numbering.  However, numbering does imply an order which most of the lesson series have....   I can't decide.  

I removed #s from the lesson names when they are linked within the lesson.  This is partially for stability over time in case lessons get re-numbered.  Plus in the text the title is more important.